runtime/compiler/env/J9VMAccessCriticalSection.hpp

```diff
@@ -64,7 +64,17 @@ class OMR_EXTENSIBLE VMAccessCriticalSection : public OMR::VMAccessCriticalSecti
             break;
 
          case tryToAcquireVMAccess:
-            _hasVMAccess = TR::Compiler->vm.tryToAcquireAccess(comp, &_acquiredVMAccess);
+#if defined(JITSERVER_SUPPORT)
```

Comment:
I see this piece of code:
```
bool
TR_J9VMBase::tryToAcquireAccess(TR::Compilation * comp, bool *haveAcquiredVMAccess)
   {
   bool hasVMAccess;
   *haveAcquiredVMAccess = false;

#if defined(JITSERVER_SUPPORT)
   // JITServer TODO: For now, we always take the "safe path" on the server
   if (comp->isOutOfProcessCompilation())
      return false;
#endif /* defined(JITSERVER_SUPPORT) */
```
so even today we'll set `_hasVMAccess = false` and `_acquiredVMAccess = false`
Do you think the new code is needed?

Reply:
The existing code in `TR_J9VMBase::tryToAcquireAccess()` does what's required at the server. The new code is not needed. Will be removed.